pkg/cmd/cmd_import.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2023 The KCL Authors. All rights reserved.
+
+package cmd
+
+import (
+	"github.com/urfave/cli/v2"
+	"kcl-lang.io/kcl-go/pkg/tools/gen"
+	"kcl-lang.io/kpm/pkg/client"
+	"kcl-lang.io/kpm/pkg/reporter"
+	"os"
+)
+
+// NewImportCmd new a Command for `kpm import`.
+func NewImportCmd(_ *client.KpmClient) *cli.Command {
+	return &cli.Command{
+		Hidden: false,
+		Name:   "import",
+		Usage:  "convert other formats to KCL file",
+		Description: `import converts other formats to KCL file.
+
+	Supported conversion modes:
+	json:            convert JSON data to KCL data
+	yaml:            convert YAML data to KCL data
+	gostruct:        convert Go struct to KCL schema
+	jsonschema:      convert JSON schema to KCL schema
+	terraformschema: convert Terraform schema to KCL schema
+	auto:            automatically detect the input format`,
+		ArgsUsage: "<file>",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "mode",
+				Aliases:     []string{"m"},
+				Usage:       "mode of import",
+				DefaultText: "auto",
+				Value:       "auto",
+			},
+			&cli.StringFlag{
+				Name:    "output",
+				Aliases: []string{"o"},
+				Usage:   "output filename",
+			},
+			&cli.BoolFlag{
+				Name:    "force",
+				Aliases: []string{"f"},
+				Usage:   "force overwrite output file",
+			},
+		},
+		Action: func(c *cli.Context) error {
+			if c.Args().Len() != 1 {
+				reporter.Report("kpm: invalid arguments")
```

Comment:
Use `reporter.ReportMsgTo` to entrue that `KpmClient` can control the output message.

```
reporter.ReportMsgTo("kpm: invalid arguments", kpmcli.GetLogWriter())
```

`kpmcli` is the parameter of `NewImportCmd(kpmcli *client.KpmClient)`.

Reply:
👌